webpack.config.js

```diff
@@ -16,8 +16,7 @@ module.exports = {
     hot: true ,
     liveReload: true,
     proxy: {
-      // '/feed': 'http://localhost:3000'
-      '/user/login' : 'http://localhost:3000'
+      '/user' : 'http://localhost:3000',
     }
```

Comment:
Might have to look into devServer proxy more because POST request for signup is currently using explicit path to localhost:3000 instead of utilizing proxy properly